docs/android/best_practices.md

```diff
@@ -0,0 +1,119 @@
+---
+title: "Android best practices"
+sidebar_label: "Best Practices"
+---
+
+## General principles
+
+In general, we should follow standard development principles such as:
+
+- **SOLID**: Single Responsibility, Open/Closed, Liskov Substitution, Interface Segregation, Dependency Inversion.
```

Comment:
Maybe add a link here? I have no idea what these principles mean, and you're explaining a lot of more common Android stuff in detail.